docs/source/guides/speed.rst

```diff
@@ -90,6 +93,15 @@ This by default comes with a performance hit, and can be disabled in most cases.
         plugins=DDPPlugin(find_unused_parameters=False),
     )
 
+.. code-block:: python
+
+    from pytorch_lightning.plugins import DDPSpawnPlugin
+
+    trainer = pl.Trainer(
+        gpus=2,
+        plugins=DDPSpawnPlugin(find_unused_parameters=False),
+    )
```

Comment:
this can be `testcode` rather, right?

Reply:
An example instead, we have internal tests.

Reply:
yes but rather keep all examples aligned with api

Reply:
`PluginsRegistry` will have a separate section with multiple DeepSpeed aliases and this. Will add it.